test/test_jit.py

```diff
@@ -7287,6 +7287,15 @@ def fn(x):
         with self.assertRaisesRegex(RuntimeError, "operation failed in interpreter"):
             fn(torch.tensor(4))
 
+    def test_trace_contiguous(self):
+        def foo(x):
+            return x[:, :, ::2].view(12).contiguous()
```

Comment:
I don't think `view` will succeed on a non-contiguous tensor, so this code is incorrect. Maybe instead of checking data pointers and relying on that behavior simply check that one of the nodes in the graph has an `aten::contiguous` kind?